packages/lib/models/Setting.ts

```diff
@@ -2345,8 +2345,8 @@ class Setting extends BaseModel {
 		return output;
 	}
 
-	public static async saveAll() {
-		if (Setting.autoSaveEnabled && !this.saveTimeoutId_) return Promise.resolve();
+	public static async saveAll(forceSaveNow = false) {
+		if (Setting.autoSaveEnabled && !this.saveTimeoutId_ && !forceSaveNow) return Promise.resolve();
```

Comment:
`saveAll()` is already a "forced save" (the non-forced one would be `scheduleSave()`). It's used a bit everywhere in the code for that purpose and works as expected as far as I know. If `saveTimeoutId_` is not set it means there's nothing to save, so forcing it wouldn't help.

I'm not sure what issue you were seeing but I suspect it's a different one.